languages/python/sqlalchemy-oso/sqlalchemy_oso/roles.py

```diff
@@ -401,9 +409,10 @@ def get_resource_users_by_role(session, resource, role_name):
     resource_name = _get_resource_name_lower(resource_model)
     filter_kwargs = {resource_name: resource, "name": role_name}
 
+    user_model_pk = getattr(user_model, inspect(user_model).primary_key[0].name)
```

Comment:
The model may have a compound primary key, which will cause this to fail.

Reply:
Why was this change needed?

Reply:
https://github.com/osohq/oso/pull/879#discussion_r638235911

This is roles v1 so not long for this world, and I believe the entirety of roles v1 assumes a non-compound PK.